frontend/src/toolbar/shared/InspectElement.js

```diff
@@ -61,19 +24,14 @@ export function InspectElement() {
                         onClick={selectingClickTargets ? selectAllElements : selectClickTargets}
                     >
                         {' '}
-                        Links Only
+                        Only Clickable
```

Comment:
I think "Only clickable" should be the only option (basically [these elements](https://github.com/PostHog/posthog-js/blob/742e71ccc05e17294f03a666fe66037425b823f8/src/autocapture-utils.js#L80)) as those are the only elements we'll have stats on anyway.

Reply:
That's fair. However I'd also then include all elements with `cursor:pointer`, as that's the standard platform-independent way of telling the browser if a random `<div>` should also be clicked or not. This is heavily used by browsers on touch devices for example...

Do clicks on divs like that actually get sent to us?

Reply:
Also, there's an annoying UX glitch when you have "only clickable"... and then nothing happens when you move your mouse in the page... as most elements won't be clickable.

To make this foolproof, I added overlays over clickable elements:

![image](https://user-images.githubusercontent.com/53387/84660769-9d84f100-af19-11ea-9a95-0aaa54dc6f95.png)

The "sparks joy" image is not a `<button>` or other linky tag. It's just a `<svg>` with `cursor:pointer`

Reply:
Yeah that's a great solution